governance_rules.go

```diff
@@ -208,16 +202,40 @@ func (r *ResponseOverride) finish() {
 }
 
 func RequestPathLookup(req *http.Request, path string) string {
+	var b io.ReadCloser
 	switch path {
 	case "request.ip_address":
 		return req.RemoteAddr
 	case "request.route":
 		return req.URL.Path
 	case "request.verb":
 		return req.Method
-	default:
-		return ""
 	}
+
+	if t := req.Header.Get("Content-Type"); path[:16] == "request.graphql." && (t == "application/json" || t == "application/graphql") {
```

Comment:
@bakennedy , we changed the path and a few other flows we should be looking at. See the notion page in the story.